src/models/linePlusBarChart.js

```diff
@@ -600,6 +627,13 @@ nv.models.linePlusBarChart = function() {
         }}
     });
 
+    chart.toggleAxis = function() {
+        switchYAxisOrder = !switchYAxisOrder;
+        var tickFormat = y1Axis.tickFormat();
+        y1Axis.tickFormat(y2Axis.tickFormat());
+        y2Axis.tickFormat(tickFormat);
+    }
+
```

Comment:
It is mostly a convenient function to make sure that the specified tickFormat for the bar and line Y-axis are corrected in case the user switches the axis. But I agree that adding this as a function is not the best way to do it. Currently I see a few options how to do it:

1. Do not set/swicht the tickFormat inside the chart at all and let the user do it manually.
2. Introduce two new variables ```barsTickFormat``` and ```linesTickFormat``` which stores the tickFormat and set them accordingly in the _y1_ or _y2_ axis, based on the option. If the user sets these variables we will use these or if he sets _y1axis.tickFormat()_ and/or _y2axis.tickFormat()_ (as you would normally do in nvd3) the new variables will be _null_ and can be initialized with _y1_ or _y2_ depending on the current state of the _switchYAxisOrder_ option.
2. Move the switching part to the setter function. But this would assume that the user sets the initial y1axis tickFormat for the bars and y2axis for the lines. 
```
    switchYAxisOrder:    {get: function(){return switchYAxisOrder;}, set: function(_){
        // Switch the tick format for the yAxis
        if(switchYAxisOrder !== _) {
            var tickFormat = y1Axis.tickFormat();
            y1Axis.tickFormat(y2Axis.tickFormat());
            y2Axis.tickFormat(tickFormat);
        }
        switchYAxisOrder=_;
    }}
```

What do you think about this?